data/members/Annabelle Feiler.json

```
@@ -0,0 +1,14 @@
+{
+  "date": "2024-01-09T00:00",
+  "name": "Annabelle Feiler,
```

```suggestion
  "name": "Annabelle Feiler",
```

Adding a missing closing quote here.